ddtrace/internal/sampling.py

```diff
@@ -208,44 +210,62 @@ def apply_span_sampling_tags(self, span):
 
 def get_span_sampling_rules():
```

Comment:
I'm usually kind of partial to breaking things up into multiple methods too, but since we're not reusing code, doesn't it make more sense to have it all in one method? @Kyle-Verhoog what do you think?

Reply:
splitting might make it easier to test. my 2c.